### ISSUE TITLE
what do you mean about face identity loss in github? 

### ISSUE THREAD

**Issue by justinday123** [anon294384/DiffuseIT#12]
hi, thank you for your work. I want to know about identity loss you wrote in github.
"For **face identity loss** when using FFHQ pre-trained model, download pre-trained ArcFace model [LINK](https://drive.google.com/file/d/1SJa5qVNM6jGZdmsnUsGNhjtrssGYuJfT/view?usp=sharing)"

**Comment by justinday123** [comment]
Also, did you used DINO-vit which is pretrained with imagenet dataset?

**Comment by cyclomon** [comment]
Hi, face identity loss is to maintain the face identity between source face image and translated face image. 
We use pre-trained face identity embedding model to implement.

Also, we used DINO-vit pre-trained with imagenet.
